test/test.ts

```diff
@@ -27,4 +33,19 @@ mocha.describe('EPub', () => {
 		var res = epub.walkNavMap(branch, [], []);
 		assert.ok(res);
 	});
+
+	mocha.it('raises descriptive errors', () => {
+		// const epub = new EPub('./example/alice.epub')
```

Comment:
what's weird is that if this line is uncommented instead of the below, the tests still pass when they shouldn't 🤔 not sure if I'm parsing it wrong or what

Reply:
Hmm, mocha doesn't wait for callbacks to be executed, does it?

Reply:
We should move to async/await... :)

Reply:
ah, you're right. I haven't written mocha tests w/ `done` for a while.

But you're also right that we should use promises, so I added a tool for awaiting events and now it works like a charm!